docs/concepts/volatiles.md

```diff
@@ -135,3 +135,4 @@ The object that is returned from the `volatile` initializer function can contain
 5.  Volatile props are preserved during the lifecycle of an instance. See also [reconciliation](reconciliation)
 6.  Changes in volatile props won't show up in the patch or snapshot stream
 7.  It is currently not supported to define getters / setters in the object returned by `volatile`
+8.  Volatile props expect real variables, (a real string, a real array, etc). Calling `create` on the types, will generate real variables.
```

Comment:
Thanks for the PR! Personally, I don't understand what "real" means in this context. Real as opposed to what? And how are they real? I actually have more questions now than before :)

Reply:
Yep, agree. How about ```
Volatile props expect data, not types. They are not directly associated with any particular type, and they can hold values of all types.
```
Is it clear?, Kindly feel free to suggest. Thanks.

Reply:
How about "Volatile prop values aren't limited to MST's types and can be assigned any value. This includes [JS primitives](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) such as `string`, `number`, `Symbol`, [Object types](https://developer.mozilla.org/en-US/docs/Glossary/Object) such as `Function`, POJOs, classes - and platform API's such as `localStorage`, `window.fetch` and basically anything you want."

Reply:
Looks good,  Did you intend ```Volatile prop values aren't limited to values of MST's types``` instead of ```Volatile prop values aren't limited to MST's types``` ?

Volatile props doesn't care about MST's type or any other type. Volatile props are like javascript variable that hold values of all types.

We are trying to make things clear for us and others. Kindly suggest. Thanks.

Reply:
Yeah, that's better :)